Add Rejoining the Beatmap Nominators requirements (#5027)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md

  - General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), as well as general behaviour in chat/forums.
  - Modding abilities: Knowledge of the general [Ranking Criteria](/wiki/Ranking_Criteria) and specific criteria of each game mode. Additional abilities like Metadata, Timing, and some others will also be taken into consideration.
- After evaluating each modder, a discussion will be made on whether the modder will join the Beatmap Nominators.

## Rejoining the Beatmap Nominators

Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with relaxed requirements. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.

### Standard rejoining requirements

Rejoining the Beatmap Nominators with standard requirements is treated similarly to failing an application.

> - The user must reapply to the Beatmap Nominators via the [website](http://bn.mappersguild.com/bnapps), with a `ModScore` of at least 0 over a period of 90 days.

Reviewer: currently returning bns who aren't "fast-tracked" need modscore>0 for 60 days. is the 90 here intentional?

Author: not to my knowledge

`m` = Minimum expected number of individual beatmaps modded in a month\
`M` = Total achieved number of individual beatmaps modded in a month

> - [The website](http://bn.mappersguild.com/bnapps) has a tool for calculating `Mod Score` on its main page.

Reviewer 2: Is this how we use backticks in the wiki? Generally it's only used for variable names.. so something like bold or quotations may work better for terms like this?

Author: The rest of the article formatted it this way, I personally have no idea or opinion on what is correct here

Reviewer 2: Would need to check the wiki style guide - should be fixed to match what we do everywhere else.

Author: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#keywords-and-commands could apply? I don't know - it is used as a keyword

  - General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), as well as general behaviour in chat/forums.
  - Modding abilities: Knowledge of the general [Ranking Criteria](/wiki/Ranking_Criteria) and specific criteria of each game mode. Additional abilities like Metadata, Timing, and some others will also be taken into consideration.
- After evaluating each modder, a discussion will be made on whether the modder will join the Beatmap Nominators.

## Rejoining the Beatmap Nominators

Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with relaxed requirements. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.

> ### Standard Rejoining Requirements

Reviewer 3: ```suggestion
### Standard rejoining requirements
```
sentence-case is mandatory

Rejoining the Beatmap Nominators with standard requirements is treated similarly to failing an application.
- The user must reapply to the Beatmap Nominators via the [website](http://bn.mappersguild.com/bnapps), with a `Mod Score` of at least 0 over a period of 90 days.
- There is a 90 day cooldown period starting from the user's removal from the Beatmap Nominators before they are able to reapply.

> - The user must provide 2 to4 recent mods in the application and retake the [Ranking Criteria](/wiki/Ranking_Criteria) test as normal.

Reviewer 3: ```suggestion
- The user must provide 2 to 4 recent mods in the application and retake the [Beatmap Nominator Test](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Nominator_Test) as normal.
```
might as well toss the relevant article here

> ### Fast-track Rejoining Requirements

Reviewer 3: ```suggestion
### Fast-track rejoining requirements
```
sentence-case is mandatory
